tq42/utils/utils.py

```diff
@@ -127,5 +126,5 @@ def get_token(service_name: str, backup_save_path: str) -> str:
             username="username",
         )
 
-    except InitError:
+    except Exception:
```

Comment:
What? Catching any Exception? Handle any exception with the same fallback? This seems very dangerous.

Reply:
Hi @rbattq, may I ask why this is dangerous? If there are any other exceptions in the prior try part of the code:

    try:
        keyring.set_password(
            service_name=service_name,
            username="username",
            password=token,
        )
        
 wouldn't it make sense to catch any possible exception and use the previous way of saving the token instead?

Reply:
I tested keyring on Mac Sonoma 14.4 and Windows 11 and it worked there however, keyring is not working on Ubuntu 20.04.6LTS by default. That is where the InitError exception was seen. Steven raised that there may be other exceptions in other Linx distros thus I proposed adding a more generic exception case handler instead.